static/app/utils/theme.tsx

```diff
@@ -902,6 +902,8 @@ export const lightTheme = {
   button: generateButtonTheme(lightColors, lightAliases),
   tag: generateTagTheme(lightColors),
   level: generateLevelTheme(lightColors),
+  stacktraceActiveBackground: lightColors.gray500,
```

Comment:
I wonder if we want this specific of a variable in our theme file 🤔 @vuluongj20 do you have any opinions on this?

Probably not worth blocking this though

Reply:
Yah, I already synced with @vuluongj20 about this already. I think for now it's okay and it's better than having that nasty .less file style the stacktrace.

Reply:
Yeah I don't like one-off color definitions like this either, but as Robin said it's the better approach out of the many worse alternatives we could think of.